src/Files.App/Strings/en-GB/Resources.resw

```diff
@@ -987,18 +987,15 @@
   <data name="SideBarEjectDevice.Text" xml:space="preserve">
     <value>Eject</value>
   </data>
-  <data name="HorizontalMultitaskingControlDuplicateTab.Text" xml:space="preserve">
+  <data name="DuplicateTab" xml:space="preserve">
     <value>Duplicate tab</value>
   </data>
   <data name="HorizontalMultitaskingControlMoveTabToNewWindow.Text" xml:space="preserve">
     <value>Move tab to new window</value>
   </data>
-  <data name="HorizontalMultitaskingControlNewTab.Text" xml:space="preserve">
+  <data name="NewTab" xml:space="preserve">
     <value>New tab</value>
   </data>
-  <data name="NewTabCtrlT" xml:space="preserve">
-    <value>New tab (Ctrl+T)</value>
-  </data>
```

Comment:
Removing this is causing the a11y issue, this key is needed for the new tabs button.

Reply:
I wonder why it wasn't caught when it was removed from the en-us file. Maybe it was falling back to this 🤔

Reply:
I would definitely not be surprised if that was the case.

Reply:
@cinqmilleans can you add this back to the new tab button?